### PR TITLE
Features: added more rules to the collection of alert rules

### PR DESCRIPTION
**Pull request by kkrumm1**
Added in the following to the Alerts Collection,

Syslog, received Alert Priority Message
Syslog, received Emergency Priority Message
Syslog, ARP Table full
APC UPS Battery Needs Replacement
APC UPS Switched to Battery Power
APC UPS in Hardware Failure Bypass Mode
APC UPS in Emergency Static Bypass Mode
APC UPS in Smart Trim Mode
HP Procurve Faulty Fan
HP Procurve Bad PSU 1
HP Procurve Bad PSU 2
HP Procurve Bad PSU 3
HP Procurve Bad PSU 4

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

---
This change is [Reviewable](https://reviewable.io/reviews/librenms/librenms/7363)

### CODE REVIEW

File: misc/alert_rules.json [hunk 1]
```diff
@@ -116,6 +116,72 @@
...
     "name": "Service critical"
   },
+  
+   {
+    "rule": "%syslog.timestamp >= %macros.past_5m && %syslog.priority ~ \"alert\"",
+    "name": "Syslog, received Alert Priority Message"
+  },
+  
+    {
+    "rule": "%syslog.timestamp >= %macros.past_5m && %syslog.priority ~ \"emerg\" || %syslog.priority ~ \"emergency\"",
```
Comment (laf) [inline misc/alert_rules.json]: `||`'s don't work outside of macros very well.

File: misc/alert_rules.json [hunk 2]
```diff
@@ -116,6 +116,72 @@
...
+    "name": "Syslog, received Emergency Priority Message"
+  },
+  
+     {
+    "rule": "%syslog.timestamp = %macros.past_5m && %syslog.msg ~ \"@arp table is full@\"",
+    "name": "Syslog, ARP table is full check on device "
+  },
+  
+   {
+    "rule": "	%sensors.sensor_type ~ \"upsAdvBatteryReplaceIndicator\" && %sensors.sensor_current = \"2\"",
```
Comment (laf) [inline misc/alert_rules.json]: Should this `~` not be a `=`

File: misc/alert_rules.json [hunk 3]
```diff
@@ -116,6 +116,72 @@
...
+    "name": "HP Procurve Bad PSU #2"
+  },
+
+   {
+    "rule": " %sensors.sensor_oid = \".1.3.6.1.4.1.11.2.14.11.1.2.6.1.4.4\" && %sensors.sensor_current = \"2\"",
+    "name": "HP Procurve Bad PSU #3"
+  },
+  
+     {
+    "rule": " %sensors.sensor_oid = \".1.3.6.1.4.1.11.2.14.11.1.2.6.1.4.5\" && %sensors.sensor_current = \"2\"",
```
Comment (laf) [inline misc/alert_rules.json]: The PSUs might be better to do `~ \".1.3.6.1.4.1.11.2.14.11.1.2.6.1.4.[2-5]\"`

Comment (kkrumm1) [reply misc/alert_rules.json]: so remove each one and make just one rule for the PSU alert rule?

Comment (laf) [reply misc/alert_rules.json]: Yup

**Comment by laf** [comment]
This is failing checks. You've got a lot of whitespace and tabs that are causing the issue.

**Pull request merged**